packages/kit/src/runtime/app/forms.js

```diff
@@ -161,14 +165,29 @@ export function enhance(form_element, submit = () => {}) {
 		let result;
 
 		try {
+			const is_valid_enctype = enctype?.match(
+				/application\/x-www-form-urlencoded|multipart\/form|text\/plain/
+			);
+
+			const headers = {
+				accept: 'application/json',
+				'Content-Type': is_valid_enctype ? enctype : 'application/x-www-form-urlencoded',
+				'x-sveltekit-action': 'true'
+			};
+
+			// @ts-expect-error
+			const body = enctype === 'multipart/form-data' ? form_data : new URLSearchParams(form_data);
```

Comment:
Why do we set the body to an instance of URLSearchParams from the form data?

Reply:
@eltigerchino  We need this data format for body being sent in an HTTP request with the 'application/x-www-form-urlencoded' content type, so we are taking the already formatted `FormData()` and converting it to `URLSearchParams()` instance. Otherwise this will not work.

Reply:
Thanks for explaining this! I learned something new.